infrastructure/azuredeploy.json

```diff
@@ -6,6 +6,14 @@
       "type": "string",
       "defaultValue": "[uniqueString(resourceGroup().id)]"
     },
+    "dev_env": {
+      "type": "string",
```

Comment:
type: "boolean" might be more applicable here

Reply:
Done.